src/Advection/weno_interpolants.jl

```diff
@@ -364,62 +371,51 @@ end
 # Calculation of WENO reconstructed value v⋆ = ∑ᵣ(wᵣv̂ᵣ)
 for buffer in [2, 3, 4, 5, 6]
     @eval begin
-        @inline stencil_sum(scheme::WENO{$buffer}, ψ, w, func, cT, val, idx, loc) = @inbounds $(metaprogrammed_stencil_sum(buffer))
+        @inline stencil_sum(scheme::WENO{$buffer}, bias, ψ, w, cT, val, idx, loc) = @inbounds $(metaprogrammed_stencil_sum(buffer))
```

Comment:
following the breadcrumbs of this argument `bias` leads here but I don't understand how this works in the end.